woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -1146,6 +1137,25 @@ def test_dataframe_methods_on_accessor_other_returns(sample_df):
         pd.testing.assert_index_equal(schema_df.ww.keys(), schema_df.keys())
 
 
+def test_dataframe_methods_on_accessor_to_pandas(sample_df):
+    if isinstance(sample_df, pd.DataFrame):
+        pytest.skip("No need to test converting pandas DataFrame to pandas")
+
+    sample_df.ww.init(name='woodwork', index='id')
+
+    if dd and isinstance(sample_df, dd.DataFrame):
+        pd_df = sample_df.ww.compute()
+        assert isinstance(pd_df, pd.DataFrame)
+        assert pd_df.ww.index == 'id'
+        assert pd_df.ww.name == 'woodwork'
```

Comment:
These assertions are the same as those below, so these can be deleted and the assertions below can be moved outside the if/else block.

Reply:
good call--moved